sechub-cli/src/mercedes-benz.com/sechub/util/tar_test.go

```diff
@@ -130,7 +130,7 @@ func TestTarFileCanBeCreated_with_exclude_patterns_applied(t *testing.T) {
 		TarWriter:   tarWriter,
 		PrefixInTar: "",
 		Folders:     []string{dirname1, dirname2},
-		Excludes:    []string{"**/sub3/**", "*.txt"},
+		Excludes:    []string{"**/sub3/**", "**/*.txt"},
```

Comment:
A `*.txt` glob pattern shouldn't match a longer path, to match an entire path, we have to add `**`.
<img width="1383" alt="image" src="https://user-images.githubusercontent.com/34098778/212327177-6623654a-b363-4222-a50d-413c661b2f9f.png">
<img width="1383" alt="image" src="https://user-images.githubusercontent.com/34098778/212327206-7b0d62f9-72c7-40fd-a30b-dac70d7f6d6b.png">